app/containers/ViewQuestion/ContentHeader.jsx

```diff
@@ -146,6 +205,69 @@ const ContentHeader = props => {
             <FormattedMessage {...messages.voteToDelete} />
           </Button>
 
+          {questionData.user === userInfo.user && (
```

Comment:
Are we allowing users to share only own question?
Users should be allowed to share any question that they like.

Reply:
This component is used for question and answer. Sharing is used only for questions by this condition and user can share any question